lib/ronad/just_one.rb

```diff
@@ -0,0 +1,7 @@
+module Ronad
+  class JustOne < Just
+    def initialize(*args)
+      super args.detect { |v| Monad === v ? v.value : v }
```

Comment:
Ronad::JustOne#initialize has the variable name 'v'

[Read more about it here.](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)